flink-ml-benchmark/src/main/resources/dct-benchmark.json

```diff
@@ -0,0 +1,40 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+  "version": 1,
+  "dct10000000": {
+    "inputData": {
+      "className": "org.apache.flink.ml.benchmark.datagenerator.common.DenseVectorGenerator",
+      "paramMap": {
+        "colNames": [
+          [
+            "input"
+          ]
+        ],
+        "seed": 2,
+        "numValues": 10000000,
+        "vectorDim": 100
+      }
+    },
+    "stage": {
+      "className": "org.apache.flink.ml.feature.dct.DCT",
+      "paramMap": {
+        "inputCol": "input",
```

Comment:
Would it be simpler to remove this line and use the default value? Same for outputCol.

Same for similar parameters used in other benchmark json files added in this PR.